README.md

```diff
@@ -179,13 +185,3 @@ Open `docker-compose.override.yml` and follow the instructions in the `cli` and
 ## Running tests (optional)
 
 `docker-compose -f docker-compose.yml -f docker-compose.tests.yml -f docker-compose.override.yml run --rm tests`
-
-## Updating devportal themes and modules
```

Comment:
Please update the PR's title because it does not just add Gulp support.

Reply:
Or move this to a new PR. This closes #22 btw.

Reply:
done